chapter06/spg/exercise1.pl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Storable qw( retrieve nstore );
+
+my $output = "stored_data.dat";
+
+my $data = {};
```

Comment:
Was going to moan about going straight to a hashref, but I suppose it means you can pass it straight into ``nstore`` later.